lib/popcorn/pop_middlewares.nit

```diff
@@ -42,23 +42,65 @@ end
 class ConsoleLog
 	super Handler
 
+	# Logger level
+	#
+	# * `1`: errors
+	# * `2`: warnings
+	# * `3`: info
+	# * `4`: debug
```

Comment:
I thought of the same thing.

PS: I hope we will have enumerated type soon… :sweat_smile: